modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,307 @@
+
+* The `image` CRD in the `caching.internal.knative.dev group` is not used by Tekton anymore, and is excluded in this release.
+
+
+[id="known-issues-1-5_{context}"]
+== Known issues
+
+* The link:https://github.com/tektoncd/catalog/tree/main/task/git-cli/0.1[git-cli] cluster task is built off the link:https://github.com/tektoncd/catalog/blob/68e44c629c9ee287393681030ed391d2c2e856cd/task/git-cli/0.1/git-cli.yaml#L32[alpine/git] base image, which expects `/root` as the user's home directory. However, this is not explicitly set in the `git-cli` cluster task.
++
+In Tekton, the default home directory is overwritten with `/tekton/home` for every step of a task, unless otherwise specified. This overwriting of the `$HOME` environment variable of the base image causes the `git-cli` cluster task to fail.
++
+This issue is expected to be fixed in the upcoming releases. For {pipelines-title} 1.5 and earlier versions, you can _use any one of the following workarounds_ to avoid the failure of the `git-cli` cluster task:
+
+** Set the `$HOME` environment variable in the steps, so that it is not overwritten.
+
+. [OPTIONAL] If you installed {pipelines-title} using the operator, then clone the `git-cli` cluster task into a separate task. This approach ensures that the operator does not overwrite the changes made to the cluster task.
+. Execute the `oc edit clustertasks git-cli` command.
+. Add the expected `HOME` environment variable to the YAML of the step:
++
+[source,yaml,subs="attributes+"]
+----
+...
+steps:
+  - name: git
+    env:
+    - name: HOME
+      value: /root
+    image: $(params.BASE_IMAGE)
+    workingDir: $(workspaces.source.path)
+...
+----
++
+[WARNING]
+====
+For {pipelines-title} installed by the operator, if you do not clone the `git-cli` cluster task into a separate task before changing the `HOME` environment variable, then the changes are overwritten during operator reconciliation.
+====
+
+** Disable overwriting the `HOME` environment variable in the `feature-flags` config map.
+
+. Execute the `oc edit -n openshift-pipelines configmap feature-flags` command.
+. Set the `disable-home-env-overwrite` flag to `true`.
++
+[WARNING]
+====
+* If you installed {pipelines-title} using the operator, then the changes are overwritten during operator reconciliation.
+
+* Modifying the default value of the `disable-home-env-overwrite` flag might break other tasks and cluster tasks, as it changes the default behavior for all tasks.
+====
+
+** Use a different service account for the `git-cli` cluster task, as the overwriting of the `HOME` environment variable happens when the default service account for pipelines is used.
+
+. Create a new service account.
+. Link your git secret to that service account.
+. Use that service account while executing a task or a pipeline.
```

Comment:
Use that service account --> Use the newly created service account

Or 'Use the service account'